src/forms/TopComponent/Views/DesktopView.tsx

```diff
@@ -27,7 +27,7 @@ const DesktopViewColumn = styled(StyledColumn)`
 	z-index: 100;
 
 	& .form-nav-wrapper {
-		margin-top: 30px;
+		margin-top: 20px;

```

Comment:
The margin-top: 20px should apply to all views that have a horizontal FormNav, not only the DesktopView.

Reply:
We should add this style to the ResponsiveView Too